Program/ConsoleLogger.cs

```diff
@@ -7,13 +7,8 @@ class ConsoleLogger : ILogger
     {
         public void Write(string message)
         {
-            if(message!= null)
+            if (message != null)
```

Comment:
To consider:

What will happen if message will be null?

Reply:
I'm not sure what you mean here - my intention was to not print anything if message is null, so the method will just return

Reply:
You didn't do anything special in case of null message formatting, plus Console.WriteLine can handle null message, so that is why I have asked

>If value is null, only the line terminator is written to the standard output stream.

https://docs.microsoft.com/en-us/dotnet/api/system.console.writeline?view=netframework-4.7.2